src/test/java/roomescape/reservation/ReservationApiTest.java

```diff
@@ -0,0 +1,93 @@
+package roomescape.reservation;
+
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class ReservationApiTest {
+
+    private static final Map<String, String> RESERVATION_BODY = new HashMap<>();
+
+    @BeforeAll
+    static void initParams() {
+        RESERVATION_BODY.put("name", "브라운");
+        RESERVATION_BODY.put("date", "2023-08-05");
+        RESERVATION_BODY.put("time", "15:40");
+    }
```

Comment:
어짜피 정해진 상수고 이후로 변경되지 않는다면, 처음부터 상수로 바로 선언해도 되지 않을까요? 
`@BeforeAll`을 활용하여 초기화해주신 이유가 있을까요? 😃 


추가적으로 이런 경우에는 [맵 팩토리](https://devbksheen.tistory.com/entry/%EB%AA%A8%EB%8D%98-%EC%9E%90%EB%B0%94-%EC%BB%AC%EB%A0%89%EC%85%98-%ED%8C%A9%ED%86%A0%EB%A6%ACCollection-Factory-%EC%86%8C%EA%B0%9C-%EB%B0%8F-%EC%82%AC%EC%9A%A9%EB%B2%95#%EB%A7%B5%20%ED%8C%A9%ED%86%A0%EB%A6%AC(Map%C2%A0Factory)-1)를 활용해봐도 좋을 것 같아요. 😃 

```suggestion
    private static final Map<String, String> RESERVATION_BODY = Map.of(
            "name", "브라운",
            "date", "2023-08-05",
            "time", "15:40"
        );
```

Reply:
해당 부분에 대하여 생각해보지 못한 것 같습니다! 알려주셔서 감사합니다!😊